modules/virt-creating-a-vm-from-a-template-with-an-attached-boot-source.adoc

```diff
@@ -6,13 +6,11 @@
 [id="virt-creating-a-vm-from-a-template-with-an-attached-boot-source_{context}"]
 = Creating a virtual machine from a template with an attached boot source
 
-After you add a boot source to a template, you can create a new virtual machine from the template.
+After you add a boot source to a template, you can create a virtual machine from the template.
 
 .Procedure
 
-. In the {product-title} web console, click *Virtualization* -> *VirtualMachines* in the side menu.
-. Click *Create* and select *Virtual Machine with Wizard*.
-. In the *Select a template* step, select an OS from the Operating System list that has the *(Source available)* label next to the OS and version name. The *(Source available)* label indicates that a boot source is available for this OS.
-. Click *Review and Confirm*.
-. Review your virtual machine settings and edit them, if required.
-. Click *Create Virtual Machine* to create your virtual machine. The *Successfully created virtual machine* page is displayed.
+. In the {product-title} web console, click *Virtualization* -> *Catalog* in the side menu.
+. Select the updated template and click *Quick create VirtualMachine*.
++
```

Comment:
Should the "+" be removed so that the final statement isn't indented with step 2?